lib/models/database/comment/comment_id_firestore.dart

```diff
@@ -1,8 +1,10 @@
 // The comment id is strong typed to avoid misuse
 import "package:flutter/foundation.dart";
+import "package:proxima/models/database/firestore/id_firestore.dart";
 
 @immutable
-class CommentIdFirestore {
+class CommentIdFirestore extends IdFirestore {
```

Comment:
I know I was not assigned to review this PR but I just noticed an important thing here:

We should avoid using inheritance here, we can use `implements` instead of `extend` and remove the constructor here:
https://github.com/ProximaEPFL/proxima/blob/6d50d25b2d9490c6b08fc8fa4c080fff917d1644/lib/models/database/firestore/id_firestore.dart#L4

Reply:
You are right !
I wasn't aware of the `implements` in dart and thought that we only had `extends`.
Thanks for suggestion, this effectively makes more sense.
Fix in fb7379bb92f99a100b1115c0d4903d2072d41d42.

Reply:
Dart has a lot of stuff when it comes to classes, even [mixins](https://dart.dev/language/mixins), be careful of the rabbit hole :)